app/views/crime_applications/index.html.erb

```diff
@@ -8,9 +8,9 @@
 </div>
 
 <%= render partial: 'shared/flash_banner' %>
-  
+
 <div class="govuk-button-group">
-  <%= button_to '#',
+  <%= button_to crime_applications_path,
     class: 'govuk-button govuk-button--start',
     data: { module: 'govuk-button' } do; t('.start_button'); end %>
```

Comment:
I think we still need to add the chevron onto the button here so it reads "Make a new application >" so that it matches the designs. 

Tiny thing though for another PR

Reply:
Yep I didn't want to do that as didn't know how far Arthur was with its PR, just needed the create request for this refactor to be completed 👍